service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceMetadataManagerTest.java

```diff
@@ -38,6 +38,8 @@ public class ControlledResourceMetadataManagerTest extends BaseUnitTest {
   private UUID resourceId;
   private ControlledResource controlledResource;
 
+  private final String readAction = SamConstants.SamControlledResourceActions.READ_ACTION;
```

Comment:
Suggest you make a static import of READ_ACTION. If you say:
```
import status SamConstants.SamControlledResourceActions.READ_ACTION;
```
you can just use `READ_ACTION` in the code.

Reply:
Thanks for the review, cleaned up the imports a bit to avoid the variable but also make the context of the constants clear.